pkg/monitor/datapath_trace.go

```diff
@@ -8,6 +8,7 @@ import (
 	"bytes"
```

Comment:
Unrelated to code: commit message says "resulting in a ~92% reduction in runtime performance", 92% reduction is actually 12.5 slower :) did you mean `92% increase`?

Reply:
Same comment about the next commit message

Reply:
Another comment about the commit message is that \`\`\` may be omitted, and you can just shift the output block by 4 characters to the right. This will be read better in the text form, and github will display it as code as well